README.md

```diff
@@ -96,6 +96,9 @@ You can also use the following default configurations:
     rpcuser=user
     rpcpass=pass
     testnet=true
+    cachehost=politeiad_user@localhost:26257/politeia
+    cacherootcert="/Users/<username>/Library/Application Support/cockroachdb/certs/ca.crt"
```

Comment:
I don't like the OSX specific dirs here at all. This should be UNIXy.

Reply:
Fixed.